cli/internal/config/config.go

```diff
@@ -27,16 +28,18 @@ func (p LLMProvider) String() string {
 
 func (p LLMProvider) IsValidLLMProvider() bool {
 	switch p {
-	case LLMProviderAnthropic, LLMProviderTesting:
+	case LLMProviderAnthropic, LLMProviderOpenAI, LLMProviderGitHubModels, LLMProviderTesting:
 		return true
 	default:
 		return false
 	}
 }
 
 const (
-	LLMProviderAnthropic LLMProvider = "anthropic"
-	LLMProviderTesting   LLMProvider = "testing"
+	LLMProviderAnthropic    LLMProvider = "anthropic"
+	LLMProviderOpenAI       LLMProvider = "openai"
```

Comment:
I'm assuming you will be documenting this in the config reference along with the other documentation called out in the change.